python/GangaLHCb/test/GPI/TestBookkeeping.py

```diff
@@ -0,0 +1,108 @@
+""" Test bookkeeping interface"""
+from __future__ import absolute_import
+
+import os
+import os.path
+
+from Ganga.Utility.logging import getLogger
+from Ganga.testlib.mark import external
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+from Ganga.Core.exceptions import PluginError
+
+logger = getLogger(modulename=True)
+
+
+@external
+class TestBookkeeping(GangaUnitTest):
+    """Test the bookkeeping interface"""
+
+    def test_path_query(self):
+        """Test that a path query works"""
+        from Ganga.GPI import BKQuery, LHCbDataset
+        bkq = BKQuery(
+            dqflag=['OK', 'UNCHECKED'],
+            path="/LHCb/Collision16/Beam6500GeV-VeloClosed-MagDown/Real Data/91000000/RAW",
+            type="Path"
+        )
+        dataset = bkq.getDataset()
+
+        assert isinstance(dataset, LHCbDataset)
+        assert len(dataset) > 0
+
+    def test_date_query(self):
```

Comment:
@egede The path you are supplying depends on the BK query type: see the examples in https://gitlab.cern.ch/lhcb-dirac/LHCbDIRAC/blob/master/LHCbDIRAC/Interfaces/API/DiracLHCb.py#L194

So for this you need to get rid of the beam conditions part of the path:
/LHCb/Collision16/Real Data/91000000/RAW

As you are requesting the whole year's data the query takes a little while - might it be worth restricting the range?

Also, what is this PR doing?